profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module

```diff
@@ -1657,3 +1656,54 @@ function multisite_drupal_toolbox_features_post_restore($op, $items) {
     }
   }
 }
+
+/**
+ * Remove PHP Warning logs from HTML5 during a loadHTML.
+ *
+ * Extend SimpleXMLElement function.
+ *
+ * @see $dom->loadHTML;
+ */
+function multisite_drupal_toolbox_load_html(&$dom, $html) {
+  $old = libxml_use_internal_errors(TRUE);
+  libxml_clear_errors();
+  if (!$dom->loadHTML($html)) {
+    return FALSE;
+  }
+  $errors = libxml_get_errors();
+  libxml_use_internal_errors($old);
+  multisite_drupal_toolbox_filter_errors($errors);
+  return TRUE;
+}
+
+/**
+ * Remove PHP Warning logs from HTML5 during a loadHTMLFile.
+ *
+ * Extend SimpleXMLElement function.
+ *
+ * @see $dom->loadHTMLFile;
+ */
+function multisite_drupal_toolbox_load_html_file(&$dom, $htmlfile) {
```

Comment:
You shouldn't pass $dom by reference here. An object is always passed by its object identifier and thus the reference is not needed. See http://php.net/manual/en/language.oop5.references.php